src/lib/haptics.ts

```diff
@@ -13,26 +13,30 @@ const hapticImpact: ImpactFeedbackStyle = isIOS
   : ImpactFeedbackStyle.Light // Users said the medium impact was too strong on Android; see APP-537s
 
```

Comment:
Could add

```js
export function useHaptics() {
  const isHapticsDisabled = useHapticsDisabled()
  return useCallback(() => {
    Haptics.default(isHapticsDisabled)
  }, [isHapticsDisabled])
}
```

Then move the callsites to

```js
const haptics = useHaptics()

// ...
haptics()
```

Reply:
nice.